src/python/turicreate/toolkits/_pre_trained_models.py

```diff
@@ -177,9 +177,21 @@ def available_parameters_subset(self, mx_params):
         ])
         return subset_params
 
+class DarkNetObjectDetectorModel(ObjectDetectorBasePreTrainedModel):
```

Comment:
Do we still need `DarkNetObjectDetectorBase` above?

That name seems more accurate, since this model is also just a base for an OD model, not a complete OD model itself